src/Xliff/Import.php

```diff
@@ -222,9 +223,13 @@ protected function handleImport(array $files): void
                     if (array_key_exists($key, $posts['post_defaults'])) {
                         $postData[$key] = $posts['post_defaults'][$key];
                     }
+                    /** @psalm-suppress UndefinedFunction **/
```

Comment:
Should add a stub in `.psalm/mlp.php` instead of this.

Reply:
Thanks for the commit @AlexP11223